### DISCUSSION TITLE
Disabilitare tap action su gauge card

### DISCUSSION THREAD

**Discussion by duedipicche** [indomus/forum#2693] [HUB: Home Assistant]
Qualcuno mi sa indicare un modo per disabilitare il tap (clik) su una gauge gard.
Gauge card non ha l'impostazione tap_action e vorrei che non si aprisse il popup dello storico quando ci clicco sopra.
Ho già inserito  il sensore negli exclude dell'history, logbook e recorder.
Grazie per l'eventuale attenzione.

**Comment by 50494554524F** [comment]
usa card_mod ed aggiungi alla definizione del gauge


```
    card_mod:

      style: |

          ha-card {

            pointer-events: none !important;

          }

```

**Comment by duedipicche** [comment]
Grazie :-)
